scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/RestartMode.java

```diff
@@ -70,6 +70,8 @@
 
     @XmlAttribute
     private String description;
+    
+    public RestartMode(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1186)